src/bin/background-worker.rs

```diff
@@ -38,7 +38,7 @@ fn main() {
     let db_url = db::connection_url(&config.db_primary_config.url);
 
     let job_start_timeout = dotenv::var("BACKGROUND_JOB_TIMEOUT")
-        .unwrap_or_else(|_| "30".into())
+        .unwrap_or_else(|_| "300".into())
```

Comment:
I think the reason this is currently so long is that it can take the background_worker several minutes to boot while it clones the index. We don't want to page someone just because a job is stuck in the queue while the worker reboots.